spec/test-data/translations/glotpress-download/stubs/fr.xml

```diff
@@ -54,4 +56,33 @@ Language: fr
         <item> - o Deux: deuxième point</item>
     </string-array>
     <string name="multi_range_statement">La première plage est 2-100 et la seconde est 123-456.</string>
+    <string name="sunday">Dimanche</string>
+    <string name="monday">Lundi</string>
+    <string name="tuesday">Mardi</string>
+    <string name="wednesday">Mercredi</string>
+    <string name="thursday">Jeudi</string>
+    <string name="friday">Vendredi</string>
+    <string name="saturday">Samedi</string>
+    <string-array name="weeks_full">
+        <item>\@string/sunday</item>
+        <item>\@string/monday</item>
+        <item>\@string/tuesday</item>
+        <item>\@string/wednesday</item>
+        <item>\@string/thursday</item>
+        <item>\@string/friday</item>
+        <item>\@string/saturday</item>
+        <item>Aucun</item>
+    </string-array>
```

Comment:
Intentional use of `\@string/` used on those `string-array/item` nodes of this GlotPress export stub, to reflect that GlotPress indeed escapes those in its exports, and to thus be able to test that we detect and auto-fix those during `quick_lint`